### PR TITLE
[TorchMLIR]add TorchMLIR converter op filter

### PR DESCRIPTION
**Pull request by Yancey1989**
related #404

### CODE REVIEW

File: pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp [hunk 1]
```diff
@@ -671,6 +671,28 @@ LogicalResult ConvertAtenOp<AtenReluOp>::matchAndRewrite(
...
+// Convert a Aten::Relu6 to HLO
+// Relu6(x) = min(AtenRelu(x), 6)
+template <>
+LogicalResult ConvertAtenOp<AtenRelu6Op>::matchAndRewrite(
+    AtenRelu6Op op,
+    OpAdaptor adaptor,
+    ConversionPatternRewriter& rewriter) const {
+  Location loc = op.getLoc();
+  Value input = adaptor.self();
+  auto inputTy = input.getType().template cast<RankedTensorType>();
```
Comment (fortianyou) [inline pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp:683]: Nit: `cast` -> `dyn_cast`, `cast` will abort the program if not succeed.

File: pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        GetTorchMlirWhiteList().end();
+  } else if (node.kind().is_prim()) {
+    auto name = c10::OperatorName(node.kind().toQualString(), "").name;
+    return GetTorchMlirWhiteList().find(name) != GetTorchMlirWhiteList().end();
+  }
+  return false;
+}
+
+std::unordered_set<std::string> GetTorchMlirWhiteList() {
+  return std::unordered_set<std::string>{
```
Comment (fortianyou) [inline pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp:36]: Nit: Shall we use a more specific white list key? Such as `aten::add(Tensor self, Tensor other, *, Scalar alpha) -> Tensor`

Comment (Yancey1989) [reply pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp:36]: maybe using the op name is okay, the ops with diff args have diff name.

File: pytorch_blade/ci_build/build_pytorch_blade.sh [hunk 3]
```diff
@@ -56,6 +56,7 @@ function ci_build() {
     TORCH_DISC_USE_TORCH_MLIR=true python3 tests/disc/ops/test_unary_ops.py
     TORCH_DISC_USE_TORCH_MLIR=true python3 tests/disc/ops/test_binary_ops.py
     TORCH_DISC_USE_TORCH_MLIR=true python3 tests/disc/ops/test_broadcast.py
```
[the remark this answers is not in the document]
Comment (Yancey1989) [reply pytorch_blade/ci_build/build_pytorch_blade.sh]: done.

File: pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp (hunk 1 above)
Comment (Yancey1989) [reply pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp]: done.

**Review by fortianyou — APPROVED** [review]
LGTM

**Pull request merged**